tests/FakeItEasy.Analyzer.CSharp.Tests/FakeItEasy.Analyzer.CSharp.Tests.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.3.1" PrivateAssets="all" />

```

Comment:
Oops. I accidentally lost the `PrivateAssets="all"`. But I notice we never had it for VisualBasic.Tests. Do you want it restored?

Reply:
Strange. It should appear as a dependency on the package, but it doesn't. No idea why...
I think it'd be better to restore it, just to be sure.

Reply:
Not sure when PrivateAssets was removed from the VB project, but it's [still there in 5.5.0](https://github.com/FakeItEasy/FakeItEasy/blob/5.5.0/src/FakeItEasy.Analyzer.VisualBasic/FakeItEasy.Analyzer.VisualBasic.csproj#L21-L22). So yes, I think it should be restored.

Reply:
> Strange. It should appear as a dependency on the package, but it doesn't. 

We are just talking about the tests. OH! I just clicked your link. One of us is talking about the tests!

I added a fixup to make test projects use PrivateAssets. I'll squash or drop as pleases you.

Reply:
Lemme walk that back. I'm still happy to hear what you think, but I've developed an opinion. We might as well keep the PrivateAssets, to more closely match the package.

Reply:
Ah, sorry. Of course, for tests it doesn't really matter. But it can't hurt either!